src/EditorFeatures/CSharpTest/Formatting/Indentation/SmartIndenterTests.cs

```diff
@@ -2570,7 +2570,7 @@ class C
                 expectedIndentation: 8);
         }
 
-        [WpfFact, Trait(Traits.Feature, Traits.Features.SmartIndent)]
+        [WpfFact(Skip = "PROTOYPE(wildcard): parse tree changed invalidating this test"), Trait(Traits.Feature, Traits.Features.SmartIndent)]
```

Comment:
Strange that VS's "find all" doesn't find this string... Oh, PROTOTYPE was misspelled.